Source/Core/Core/MemoryWatcher.h

```diff
@@ -37,8 +43,13 @@ class MemoryWatcher final
 
   bool m_running;
 
+#ifdef _WIN32
+  HANDLE m_pipe;
```

Comment:
It's not optimal, but you can probably cast the `HANDLE` to a `void*` here to avoid including `Windows.h` in the header.